README.md

```diff
@@ -1,83 +1,134 @@
+*   **Persistent Policy Enforcement:** Access policies are bound to the data, allowing data owners or administrators to manage access even after the data has been shared outside organizational boundaries.
+*   **End-to-End Auditability:** The protocol facilitates comprehensive logging of key requests, providing a reliable audit trail of data access attempts.
+*   **Large File & Streaming Support:** Efficiently handles large data objects through secure streaming mechanisms, maintaining integrity throughout the process.
+*   **Policy Integrity:** Cryptographically binds the access policy defined in the manifest to the key access information, preventing policy tampering after creation.
+*   **Offline Creation:** Allows TDF objects to be created securely by clients even without immediate network connectivity to a key server, thanks to policy binding assurances.
+*   **Federated Key Management:** Supports scenarios where multiple Key Access Servers (KAS), potentially hosted by different organizations, can collaboratively manage access to a single TDF object, enabling secure cross-domain collaboration in a zero-trust manner.
 
-### 2. Policy Binding and Offline Create
-With embedding cryptographically bound policy and wrapped keys, we enable a high assurance key server. 
+## Security Principles: The C.I.A. Triad
 
-![offline](https://files.readme.io/f5fb283-Screen_Shot_2018-12-10_at_9.15.27_AM.png "Offline create")
+OpenTDF is designed with the fundamental security principles of Confidentiality, Integrity, and Availability (C.I.A.) at its core:
 
-### 3. Key Server Federation
-Want to protect files such that two (or more) organizations control the keys? It is now possible with TDF3. [keyAccess](schema/tdf/KeyAccessObject.md) object in particular allows for array of objects, which can allow for multiple KAS servers to participate in an object key grant.
+*   **Confidentiality:** Ensures that sensitive data is only accessible to authorized users through strong encryption and attribute-based access control.
+*   **Integrity:** Maintains data authenticity and prevents unauthorized modifications through cryptographic binding of policies and payloads.
+*   **Availability:** Enables secure access to protected data through distributed key management and offline creation capabilities.
+
+These principles work together to provide comprehensive data protection while maintaining usability and accessibility for authorized users.
+
+## TDF Structure
+
+By default, a TDF object is packaged as a standard Zip archive file, typically using the `.tdf` extension appended to the original filename. This archive contains two primary components: 
+
+1. **`manifest.json`:** The metadata manifest described in the Key Concepts section. It holds instructions for decryption and access control. 
+2. **`payload`:** The encrypted data payload itself. 
+
+However, a TDF can be encoded in other ways. For example, as an HTML document:
```

Comment:
is this "true"? like technically someone could hand roll it but do we want to mention html at all, since i dont think opentdf supports ti directly?